sinch/domains/voice/__init__.py

```diff
@@ -0,0 +1,389 @@
+)
+from sinch.domains.voice.models.calls.requests import (
+    GetVoiceCallRequest,
+    UpdateVoiceCallRequest,
+    ManageVoiceCallRequest,
+    Action
+)
+from sinch.domains.voice.models.calls.responses import (
+    GetVoiceCallResponse,
+    UpdateVoiceCallResponse,
+    ManageVoiceCallResponse
+)
+from sinch.domains.voice.models.conferences.requests import (
+    GetVoiceConferenceRequest,
+    KickAllVoiceConferenceRequest,
+    KickParticipantVoiceConferenceRequest,
+    ManageParticipantVoiceConferenceRequest
+)
+from sinch.domains.voice.models.conferences.responses import (
+    GetVoiceConferenceResponse,
+    KickAllVoiceConferenceResponse,
+    ManageParticipantVoiceConferenceResponse,
+    KickParticipantVoiceConferenceResponse
+)
+from sinch.domains.voice.models.applications.requests import (
+    AssignNumbersVoiceApplicationRequest,
+    UnassignNumbersVoiceApplicationRequest,
+    QueryNumberVoiceApplicationRequest,
+    UpdateCallbackUrlsVoiceApplicationRequest,
+    GetCallbackUrlsVoiceApplicationRequest
+)
+from sinch.domains.voice.models.applications.responses import (
+    GetNumbersVoiceApplicationResponse,
+    AssignNumbersVoiceApplicationResponse,
+    UnassignNumbersVoiceApplicationResponse,
+    GetCallbackUrlsVoiceApplicationResponse,
+    QueryNumberVoiceApplicationResponse
+)
+
+
+class Callouts:
+    def __init__(self, sinch):
+        self._sinch = sinch
+
+    def text_to_speech(
+        self,
+        destination: Destination,
+        cli: str = None,
+        dtmf: str = None,
+        domain: str = None,
+        custom: str = None,
+        locale: str = None,
+        text: str = None,
+        prompts: str = None,
+        enable_ace: bool = None,
+        enable_dice: bool = None,
+        enable_pie: bool = None
+    ) -> VoiceCalloutResponse:
+        return self._sinch.configuration.transport.request(
+            CalloutEndpoint(
+                callout_method=CalloutMethod.TEXT_TO_SPEECH.value,
+                request_data=TextToSpeechVoiceCalloutRequest(
+                    destination=destination,
+                    cli=cli,
+                    dtmf=dtmf,
+                    domain=domain,
+                    custom=custom,
+                    locale=locale,
+                    text=text,
+                    prompts=prompts,
+                    enableAce=enable_ace,
+                    enableDice=enable_dice,
+                    enablePie=enable_pie
+                )
+            )
+        )
+
+    def conference(
+        self,
+        destination: Destination,
+        conference_id: str,
+        cli: str = None,
+        conference_dtmf_options: ConferenceDTMFOptions = None,
+        dtmf: str = None,
+        conference: str = None,
+        max_duration: int = None,
+        enable_ace: bool = None,
+        enable_dice: bool = None,
+        enable_pie: bool = None,
+        locale: str = None,
+        greeting: str = None,
+        moh_class: str = None,
```

Comment:
REST API documentation specifies this property as "mohClass". I prefer to stick to the REST API spec

Reply:
I may not have been clear, sorry: the specification details the valid values for this property. I was suggesting to use them instead of a wildcard string: this way the SDK will event be closer to the REST API spec